CHANGELOG.md

```diff
@@ -21,7 +21,12 @@ Enables the use of pre-release builds of Omnisharp. Downloading a pre-release bu
 
 #### Editor
 
-* Updated the Omnisharp Status Bar to appear on left instead of right because it displays the project level info(which is a characteristic of the elements on the left side of the window) and removed ambiguities in its behavior.([#2146](https://github.com/OmniSharp/omnisharp-vscode/issues/2146), PR: [@2133](https://github.com/OmniSharp/omnisharp-vscode/pull/2133))
+* Splits the OmniSharp status bar in two parts both of which appear on the left and have specific responsibilities. ([#2146](https://github.com/OmniSharp/omnisharp-vscode/issues/2146), PR: [@2133](https://github.com/OmniSharp/omnisharp-vscode/pull/2133))
```

Comment:
Note that "status bar" is the wrong terminology. The "status bar" is the entire strip of information at the bottom of VS Code. The individual bits of information are "status bar items".